swagger_meta.go

```diff
@@ -18,7 +18,7 @@
 //	- application/json
 //
 //	SecurityDefinitions:
-//	oryAccessToken:
+//	APIKey:
```

Comment:
This needs some proper checking with the generated SDKs to see whether it's a breaking change. And please keep the existing semantics:

```suggestion
//	oryApiKey:
```

Reply:
I've re-done this in a new PR, would you mind taking a look? 
https://github.com/ory/kratos/pull/3109